### DISCUSSION TITLE
关于部分可能的冗余代码的问题

### DISCUSSION THREAD

**Discussion by Adentent** [MCSLTeam/MCSL2#189] [提问区域]
很抱歉打扰各位开发者.

我十分感谢这个项目, 这为我的开服提供了莫大的便利. 但在我阅读了部分相关代码之后, 我发现这其中貌似存在少部分的冗余代码 `[ 我所见的已经存在了一年, 在项目中 ( 据我所看到的 ) 从未被调用 ]`, 具体见 [这个 commit](https://github.com/Adentent/MCSL2/commit/a9386e56c57ee95350ac7baa33a3c80aa30bd5c7). 因此我想提出一个疑问, 即:

1. 这些代码是否真的属于冗余代码, 并且极大概率不会再被使用?
2. 如果不是, 那么这些代码的用途, 以我短见, 应当被注明.

感谢您的阅读, 我很高兴能使用如此优秀的软件, 在此向各位 contributors 致以崇高的敬意.

**Comment by lxhtt** [comment]
有人挖坑，我不说
@wyx55520

**Reply by lxhtt** [reply]
他负责的这一块  然后弃坑😅

**Comment by kitUIN** [comment]
看样子挖的还是插件大坑(

**Comment by Adentent** [comment]
原来是这样（）

感觉挺幽默的（划）
